docs/continuous-testing/running-android-unit-tests.md

```diff
@@ -124,6 +124,27 @@ If you're using UI tests with emulators, you must select an Intel device (**Defa
 
 :::
 
+### Jacoco Test Coverage
+
+If you are using Jacoco tool for test coverage in the project, you can get coverage percentage from the test reports generated by Jacoco. There are 6 different types of coverage percentage calculation. The coverage percentage will be calculated according to the selected value
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/be2556-jacocoCoverage2.png" />
+
+:::info
+Ranked from the most detailed coverage percentage to the most general coverage percentage.
+
+- **INSTRUCTION**: The smallest unit JaCoCo counts are single Java byte code instructions.
+- **BRANCH**: JaCoCo also calculates branch coverage for all if and switch statements.
+- **COMPLEXITY**: JaCoCo also calculates cyclomatic complexity for each non-abstract method and summarizes complexity for classes, packages and groups.
+- **LINE**: For all class files that have been compiled with debug information, coverage information for individual lines can be calculated.
+- **METHOD**: A method is considered as executed when at least one instruction has been executed. As JaCoCo works on byte code level also constructors and static initializers are counted as methods.
```

Comment:
```suggestion
- **METHOD**: JaCoCo considers a method as executed when at least one instruction has been executed. Since JaCoCo works at the bytecode level, it counts constructors and static initializers as methods.
```